spec/lex.dd

```diff
@@ -130,8 +130,15 @@ $(GNAME Comment):
     $(GLINK LineComment)
     $(GLINK NestingBlockComment)
 
+$(P In lex language, in order to concatenate, + is used instead of $(OPT) for a block comment.)
+$(P The given example with 2  $(D */)  $(D */) one following the other one will generate an error because the
+syntax is incorrect, there is an invalid sequence: the comment stops at the first  $(D */), and the other one
+is left outside. There can't be used all the Unicode characters because errors are generated depending on
+the chosen character. In this case, because a comment is represented by a text between $(D /**/), then when a third
+one appears, the code doesn't know what it means and will be outside of the comment section. )
```

Comment:
This paragraph contains grammar mistakes and is overly complicated. Please try to simplify.